spreadsheet/context.go

```diff
@@ -131,3 +142,33 @@ func (e *evalContext) GetWidth(colIdx int) float64 {
 	}
 	return 0
 }
+
+// GetEpoch returns a workbook's time epoch.
+func (e *evalContext) GetEpoch() time.Time {
+	return e.s.w.Epoch()
+}
+
+var dbcs []string = []string{
```

Comment:
what does dbcs stand for?

Reply:
added description

Reply:
Please use standard convention in comments.  Space between / and text. And sentences should always end with a dot.

Reply:
fixed